File: web3rpc/rpc-specs/code-samples/javascript/klay/miscellaneous/getActiveAddressFromRegistry.js
const { JsonRpcProvider } = require("@kaiachain/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-kairos.node.kaia.io");
  const kipType = "kip113";
  const blockNumber = "latest";

  provider.klay
    .getActiveAddressFromRegistry(
      kipType,
      blockNumber,
      {},
      (err, data, response) => {}
    )
    .then((data) => {
      console.log(data);
    });
})();

File: web3rpc/rpc-specs/code-samples/python/klay/miscellaneous/getActiveAddressFromRegistry.py
from web3 import Web3
from web3py_ext import extend

host = "https://public-en-kairos.node.kaia.io"

name= 'kip14'
blockNumber = 'latest'

w3 = Web3(Web3.HTTPProvider(host))
klay_response = w3.klay.get_active_address_from_registry(name, blockNumber)

print(klay_response)
